channels/fr.m3u

```diff
@@ -145,6 +145,8 @@ http://51.195.88.12:4050/live/euronewsmobpp2/playlist.m3u8
 https://rakuten-euronews-5-de.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="EuronewsenEspanol.fr" tvg-name="Euronews en Español" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://i.imgur.com/K6bo1R3.jpg" group-title="News",Euronews en Español (720p)
 https://rakuten-euronews-4-es.samsung.wurl.com/manifest/playlist.m3u8
+#EXTINF:-1 tvg-id="EuronewsEnEspanol.fr" tvg-name="Euronews en Español" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://i.imgur.com/K6bo1R3.jpg" group-title="News",Euronews en Español (YouTube)
```

Comment:
There is no need to specify the source of broadcast in the name of the channel.

Reply:
Fixed.

Reply:
Fixed.